### DISCUSSION TITLE
Enhancement : Adding a data import option by default

### DISCUSSION THREAD

**Discussion by Avin-Mathew** [jowilf/starlette-admin#479] [Ideas]
I have been using starlette-admin for some time, and I would really like to see an option in it to import data from csv and other formats, just like it exports data. Do add this at the earliest. Thanks Avin

**Comment by hasansezertasan** [comment]
Thank you @Avin-Mathew for the recommendation. Do you have any alternative or anything that can resemble what you want?

**Comment by Avin-Mathew** [comment]
I was thinking of the import functionality like django-import-export package (https://pypi.org/project/django-import-export/) that when used along with django can be used for importing data from a variety of formats.
